__integrations__/environments/env.base

```diff
@@ -0,0 +1,28 @@
+COSMOSDB_PORT=3000
+COSMOSDB_URI=https://cosmosdb:${COSMOSDB_PORT}/
+COSMOSDB_KEY=dummykey
+COSMOSDB_NAME=integrationtests
+
+REDIS_URL=redis
+REDIS_TLS_ENABLED=false
+
+FUNCTION_PORT=7071
+USE_FALLBACK=false
+FF_TYPE=none
+
+STORAGE_CONN_STRING=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://storage-account:10000/devstoreaccount1;QueueEndpoint=http://storage-account:10001/devstoreaccount1;TableEndpoint=http://storage-account:10002/devstoreaccount1;
+DF_STORAGE_CONN_STRING=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://storage-account:10000/devstoreaccount1;QueueEndpoint=http://storage-account:10001/devstoreaccount1;TableEndpoint=http://storage-account:10002/devstoreaccount1;
```

Comment:
do we need this?

Reply:
No, thanks. Removed in [[#IOCIT-357] remove unused variable](https://github.com/pagopa/io-functions-lollipop/pull/9/commits/1c0e47cac14e46233c98e210561641bd7006e709)